src/controllers/rankings.js

```diff
@@ -0,0 +1,66 @@
+const { graphql } = require('graphql');
+const { schema } = require('./graphql')
+
+const query = `
+query UserRankings($user_id: Int) {
+  userRankings(prev_ranking_idIsNull: true, user_id: $user_id) {
+    ...RankingFields
+    ...RankingsRecursive
+  }
+}
+
+fragment RankingsRecursive on UserRankings {
+  nextRanking {
+    ...RankingFields
+    nextRanking {
+      ...RankingFields
+      nextRanking {
+        ...RankingFields
+        nextRanking {
+          ...RankingFields
+          nextRanking {
+            ...RankingFields
+            nextRanking {
+              ...RankingFields
+              nextRanking {
+                ...RankingFields
+                nextRanking {
+                  ...RankingFields
```

Comment:
I'll put a vote in for parameterizing this as 
```javascript
const generateRankingsRecursive = (n) => {
    if (n === 1) {
        return 'nextRanking { ...RankingFields }'
    } else {
        return `nextRanking { ...RankingFields ${generateRankingsRecursive(n-1)} }`
    }
```
Something like that. We don't really need it, it just might make it easier to grow/shrink this list.

Reply:
I think we do need it